apps/server/src/modules/common-cartridge/export/elements/v1.1.0/common-cartridge-metadata-element.ts

```diff
@@ -0,0 +1,42 @@
+import { CommonCartridgeElementType, CommonCartridgeVersion } from '../../common-cartridge.enums';
+import { CommonCartridgeElement } from '../../interfaces/common-cartridge-element.interface';
+
+export type CommonCartridgeMetadataElementPropsV110 = {
+	type: CommonCartridgeElementType.METADATA;
+	version: CommonCartridgeVersion;
+	title: string;
+	creationDate: Date;
+	copyrightOwners: string[];
+};
+
+export class CommonCartridgeMetadataElementV110 extends CommonCartridgeElement {
+	constructor(private readonly props: CommonCartridgeMetadataElementPropsV110) {
+		super(props);
+	}
+
+	public getSupportedVersion(): CommonCartridgeVersion {
+		return CommonCartridgeVersion.V_1_1_0;
+	}
+
+	public getManifestXmlObject(): Record<string, unknown> {
+		return {
+			schema: 'IMS Common Cartridge',
+			schemaversion: '1.1.0',
+			'mnf:lom': {
```

Comment:
Do we have a type / interface behind it that can we use?
If not i think self define will also help to replace Record<string, unknown> with more specifc type.

The same for all the other getManifestXmlObject() 
` extends CommonCartridgeElement<MyInterface>`